src/lib/snark_worker_lib/debug.ml

```diff
@@ -10,26 +29,19 @@ module Inputs = struct
     let worker_wait_time = 0.1
   end
 
-  module Proof = struct
-    (* TODO : version *)
-    type t =
-      Transaction_snark.Statement.Stable.V1.t
-      * Coda_base.Sok_message.Digest.Stable.V1.t
-    [@@deriving bin_io, sexp]
-  end
-
-  module Statement = Transaction_snark.Statement
+  module Statement = Transaction_snark.Statement.Stable.V1
 
   module Public_key = struct
     include Signature_lib.Public_key.Compressed
 
     let arg_type = Cli_lib.Arg_type.public_key_compressed
   end
 
-  module Transaction = Coda_base.Transaction
-  module Sparse_ledger = Coda_base.Sparse_ledger
-  module Pending_coinbase = Coda_base.Pending_coinbase
+  module Transaction = Coda_base.Transaction.Stable.Latest
```

Comment:
see the comment under `prod.ml`: choosing particular inputs creates a new version.